src/lib.rs

```diff
@@ -94,11 +102,28 @@ impl Display for NotifyState<'_> {
             NotifyState::FdStoreRemove => write!(f, "FDSTOREREMOVE=1"),
             #[cfg(feature = "fdstore")]
             NotifyState::FdName(name) => write!(f, "FDNAME={}", name),
+            NotifyState::MonotonicUsec(usec) => write!(f, "MONOTONIC_USEC={}", usec),
             NotifyState::Custom(state) => write!(f, "{}", state),
         }
     }
 }
 
+impl NotifyState<'_> {
+    /// Create a new [`NotifyState::MonotonicUsec`] using the current system monotonic time.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use sd_notify::NotifyState;
```

Comment:
This didn't build (`no_run` is "don't run", not "don't build"), but should be fine to run.

Reply:
Not sure what you mean?

```
cargo test
...
test src/lib.rs - NotifyState<'_>::monotonic_usec_now (line 116) ... ok
```

Reply:
I had to add the `use`, otherwise:

```
---- src/lib.rs - NotifyState<'_>::monotonic_usec_now (line 116) stdout ----
error[E0433]: failed to resolve: use of undeclared type `NotifyState`
 --> src/lib.rs:118:9
  |
4 | let _ = NotifyState::monotonic_usec_now().expect("Failed to read monotonic time");
  |         ^^^^^^^^^^^ use of undeclared type `NotifyState`
  |
help: consider importing this enum
  |
3 + use sd_notify::NotifyState;
  |
```

And we can remove `no_run` since it doesn't do anything bad.

Not sure how it passed CI, maybe it didn't run?